privacy-dashboard/privacy-dashboard-impl/src/test/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardRendererTest.kt

```diff
@@ -82,7 +84,7 @@ class PrivacyDashboardRendererTest {
 
         testee.render(aViewState())
 
-        verify(spyWebView, times(6)).evaluateJavascript(captor.capture(), eq(null))
+        verify(spyWebView, times(7)).evaluateJavascript(captor.capture(), eq(null))
```

Comment:
can you also add the call that is triggered?

I assume:

`assertNotNull(captor.allValues.find { it.startsWith("javascript:onChangeConsentManaged") })`

Reply:
Yup, added it in the last commit